CosmosDBDotnetTemplates.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>1.0.0</PackageVersion>
+    <PackageId>Microsoft.Azure.Cosmos.Templates</PackageId>
+    <Title>Cosmos DB Dotnet Templates</Title>
+    <Authors>Cosmos DB</Authors>
+    <Description>Dotnet templates for Cosmos DB.</Description>
+    <PackageTags>Cosmos-DB;WebAPI,API-for-MongoDB</PackageTags>
+
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IncludeContentInPack>true</IncludeContentInPack>
```

Comment:
CPU x64?

Reply:
I'm not sure what you mean by this, do you think I should declare a target platform?

Reply:
Yes, the target platform. I guess it doesn't matter for mongo scenarios.